redaxo/src/addons/be_style/plugins/customizer/assets/js/main.js

```diff
@@ -1,18 +1,15 @@
-$(document).on('pjax:end',   function(){
-    Customizer.init();
-});
-$(document).ready(function(){
-    Customizer.init();
+$(document).on('rex:init', function (event, container) {
```

Comment:
Ich bin auf dem Gebiet absolut kein Experte, aber ich verstehe das so, dass "ready.rex" auch bei "ready" ausgelöst werden würde. Man kann nur eben die Eventhandler des Namespaces leichter wieder löschen etc.
(Die Beispiele aus dem Link würden doch auch bei "click" ausgelöst werden, oder nicht?)

https://learn.jquery.com/events/introduction-to-custom-events/
Hier wird auch die Doppelpunkt-Notation für eigene Events verwendet.